bin/deploy-bamboo.sh

```diff
@@ -62,10 +62,10 @@ tmp
 EOF
 
 cat <<EOF > Dockerfile
-FROM node:14.18
+FROM node:18.16-bullseye
 COPY . /build
 WORKDIR /build
-RUN npm ci --production && npm run build
+RUN npm ci --omit=dev && npm run build
```

Comment:
`--omit=dev` is not used instead of `--production `to prevent installing `devDependencies` https://stackoverflow.com/questions/9268259/how-do-you-prevent-install-of-devdependencies-npm-modules-for-node-js-package